src/services/exposed/scope-service.js

```diff
@@ -0,0 +1,13 @@
+const context = require('../../context');
+
+class ScopeService {
+  static async getScopes(renderingId) {
+    const { configStore, forestServerRequester } = context.inject();
+    const { envSecret } = configStore.lianaOptions;
+    const queryParams = { renderingId };
+
+    return forestServerRequester.perform('/liana/scopes', envSecret, queryParams);
+  }
+}
```

Comment:
In order to complete the feature, this service should be able to provide a filter which has the same format as the filters which are used for segments, charts, and so on.

This service should perform the substitution of $currentUser.* entries

```
// this._model == reviews sequelize model
// user == user from request.user (it contains the renderingId)
await getScope(this._model, user) == {
   "aggregator":"and",
   "conditions":[
       {"field":"book:id","operator":"equal","value":2},
       {"field":"id","operator":"equal","value":3}
    ]
}
```